src/components/Navbar.js

```diff
@@ -34,6 +34,9 @@ const Navbar = () => {
               Profile
             </Button>
             <Logout />
+            <Button color="inherit" component={Link} to="/wishlist">
+              Wishlist
+            </Button>
```

Comment:
Not a huge deal, but usually websites have the `Logout` button as the last (right-most) entry in the navbar so it's easy to find.

Reply:
You are right, I moved the logout to end